src/components/Footer/Footer.tsx

```diff
@@ -2,19 +2,39 @@ import * as Styled from './styles'
 
 import LogoIcon from '../../../public/logo.svg'
 import Markdown from '../Markdown/Markdown'
+import Image from 'next/image'
 
 export default function Navbar() {
   return (
     <Styled.ColorBar>
       <Styled.Footer>
         <Styled.LogoWrapper>
-          <LogoIcon alt='Climate Compatible Growth logotype' />
-          <p>
-            {`Copyright © Climate Compatible Growth ${new Date().getUTCFullYear()}. All rights reserved.`}
-          </p>
+          <Styled.LogoInnerWrapper>
+            <LogoIcon alt='Climate Compatible Growth logotype' />
+            <p>
+              {`Copyright © Climate Compatible Growth ${new Date().getUTCFullYear()}. All rights reserved.`}
+            </p>
+          </Styled.LogoInnerWrapper>
         </Styled.LogoWrapper>
         <Styled.ContactWrapper>
           <Markdown>{footerContent}</Markdown>
+          <br />
```

Comment:
Could/should not these be part of the markdown instead? `\n\n` is roughly equivalent to `<br>`, links look like `[human-readable-text](https://actual.url.com/hello-world)`, and images something similar. Give it a try, and talk to me if something seems weird or bad.